9.2/Dockerfile.rhel7

```diff
@@ -46,6 +43,9 @@ RUN yum install -y yum-utils gettext && \
     test "$(id postgres)" = "uid=26(postgres) gid=26(postgres) groups=26(postgres)" && \
     chmod -R go+rwx /var/lib/pgsql && chmod go+w /var/run/postgresql
 
+COPY run-*.sh /usr/local/bin/
+COPY contrib /var/lib/pgsql/
```

Comment:
i don't think you can do this..  you're chmod'ing the directory before you copy stuff into it now, which means the things you copy into it are not getting chmod'd.

you *could* break the RUN up to do the chmod after, but that adds layers and build time.  

how i've handled this in the past is to reorder it while i'm working on the image/scripts, but then put it back before committing the changes.  i know that's a bit of a pain, but it maintains a cleaner dockerfile.